src/Controller/UI/DatalandController.php

```diff
@@ -1,65 +1,102 @@
 <?php
 
-namespace Pelagos\Bundle\AppBundle\Controller\UI;
+namespace App\Controller\UI;
 
-use Pelagos\Exception\InvalidGmlException;
+use App\Handler\EntityHandler;
+use App\Util\DataStore;
+use App\Util\Geometry;
+use App\Util\Metadata;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 use Symfony\Component\HttpFoundation\BinaryFileResponse;
 use Symfony\Component\HttpFoundation\RedirectResponse;
 use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\HttpKernel\Exception\BadRequestHttpException;
 use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
 use Symfony\Component\Security\Core\Exception\AccessDeniedException;
 
-use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Symfony\Component\Routing\Annotation\Route;
 
-use Pelagos\Entity\Dataset;
-use Pelagos\Entity\DIF;
-use Pelagos\Entity\DatasetSubmission;
-use Pelagos\Entity\Metadata;
-use Pelagos\Util\GmlUtil;
+use App\Exception\InvalidGmlException;
+
+use App\Entity\Dataset;
+use App\Entity\DatasetSubmission;
+use App\Util\GmlUtil;
 
 /**
  * The Dataset Monitoring controller.
- *
- * @Route("/data")
  */
-class DatalandController extends UIController
+class DatalandController extends AbstractController
 {
+    /**
```

Comment:
I like to use consistent attribute ordering as they are used, but I guess it's not really a big deal. What do y'all thank?

Reply:
That's probably something we should start doing, for now i'm ok with it. But if you really wanna re-order it, that's ok too. These are all new. I wouldn't do it with old ones. For now I think leave well enough alone.